tests/modules/sightings/resources/utils.py

```diff
@@ -83,6 +83,21 @@ def read_sighting(flask_app_client, user, sight_guid, expected_status_code=200):
     return response
 
 
+def update_sighting(
```

Comment:
For consistency, every other module utils has this as patch_{module} rather than update_{module}
Also as the patch_data is the variable bit and expected_status_code is the likely stable bit, patch_data should come first. 
That would make some of the calls simpler as they wouldn't need the named parameters

Reply:
i flopped the order of the last 2 args here and the create_ one you brought up on colins PR.  :+1:  thanks for the tip here.

i can change the name too.  i naively assumed we were mapping the http-method to instead be its CRUD value, e.g. POST => create; GET => read.  so why not PATCH => update?

but i will go with the _other_ consistency (i.e. consistent across files, not across methods), haha, and rename this to patch!  :)